### PR TITLE
Feature/config file

### PR DESCRIPTION
**Pull request by meganrm**
Problem
=======
Pull out run time configs from the recipe docs and into a config file. #47
Also see the database wiki. 

Solution
========
Created a config file format that holds older recipe values and some of our new command line values. 
I made two default configs, one that has settings for viewing the grid and one that is more of a default pack config. 

## Type of change
Please delete options that are not relevant.

* New feature (non-breaking change which adds functionality)
* This change requires a documentation update
* This change requires updated or new tests

Steps to Verify:
----------------
We have our first tests ! 🎉
You can also now run `pack --recipe cellpack/test-recipes/test_compartment.json --config packing-configs/run.json`

Screenshots (optional):
-----------------------
Show-n-tell images/animations here

Keyfiles (delete if not relevant):
-----------------------
1. pack.py

Thanks for contributing!

### CODE REVIEW

File: cellpack/autopack/binvox_rw.py [hunk 1]
```diff
@@ -285,7 +285,7 @@ def read(fp):
     #  The first byte of each pair is the value byte and is either 0 or 1
```
Comment (meganrm) [inline cellpack/autopack/binvox_rw.py:285]: everything here is fixing pytest warnings

File: docs/RECIPE_SCHEMA.md [hunk 2]
```diff
@@ -115,24 +115,17 @@ _Optional `string[]`_
 An array that defines the names of directional gradients to use.
 
 ### innerGridMethod
-_Optional enum_. One of `"bhtree"`, `"jordan"`, `"sdf"`, `"jordan3"`, `"pyray"`, `"floodfill"`, `"binvox"`, `"trimesh"`, `"scanline"`. Default: `"jordan3"`.
+_Optional enum_. One of `"bhtree"`, `"raytrace"`, `"sdf"`, `"pyray"`, `"floodfill"`, `"binvox"`, `"trimesh"`, `"scanline"`. Default: `"jordan3"`.
```
Comment (mogres) [inline docs/RECIPE_SCHEMA.md:118]: Is the default still jordan3?

**Review by mogres — COMMENTED** [review]
Thanks for adding these features! The recipe/config loader setups look great.

**Review by mogres — APPROVED** [review]

**Comment by mogres** [comment]
Note: the simulariumio cellpack converter should be updated to reflect the change from `cellpack.autopack.iotools_simple` to `cellpack.autopack.loaders.recipe_loader`

**Pull request merged**